lib/src/utils/serialization_utils.dart

```diff
@@ -41,7 +42,7 @@ import 'package:ledger_cardano/src/utils/validation_exception.dart';
 import 'package:ledger_flutter/ledger_flutter.dart';
 import 'dart:convert';
 
-typedef SerializeTokenAmountFn<T> = Uint8List Function(T value);
+typedef SerializeTokenAmountFn = Uint8List Function(BigInt value);
```

Comment:
`SerializationUtils.int64ToBuf` is always used when `SerializeTokenAmountFn` is needed. 

`SerializeTokenAmountFn` should be deleted entirely and not passed as argument.

Reply:
done